test/unit/browsered/field_validation_checks_test.js

```diff
@@ -41,4 +47,85 @@ describe('field validation checks', () => {
       expect(isFieldGreaterThanMaxLengthChars('123456ABC', 'que')).to.equal(false)
     })
   })
+  describe('isNotValidAccountNumber', () => {
+    it('should validate successfully for 8 digits', () => {
+      const accountNumber = '00012345'
```

Comment:
As somebody with a bank account number that begins with a zero and who has been bitten by shoddy websites before, I thank you! 🙇🏼‍♂️